opencti-platform/opencti-front/src/private/Index.jsx

```diff
@@ -25,20 +25,17 @@ import Message from '../components/Message';
 import { BoundaryRoute, NoMatch } from './components/Error';
 import StixObjectOrStixRelationship from './components/StixObjectOrStixRelationship';
 import SearchBulk from './components/SearchBulk';
-import TopBar from './components/nav/TopBar';
 import RootCases from './components/cases/Root';
 import SystemBanners from '../public/components/SystemBanners';
 import TimeoutLock from './components/TimeoutLock';
 import useAuth from '../utils/hooks/useAuth';
-import SettingsMessagesBanner, {
-  useSettingsMessagesBannerHeight,
-} from './components/settings/settings_messages/SettingsMessagesBanner';
+import SettingsMessagesBanner, { useSettingsMessagesBannerHeight } from './components/settings/settings_messages/SettingsMessagesBanner';
+import TopBar from "@components/nav/TopBar";
```

Comment:
is this notation `@` supposed to be used for imports from now on?

Reply:
It will, but it's not mandatory to do something about it.